x-pack/test/functional/apps/enterprise_search/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ loadTestFile }: FtrProviderContext) {
+  describe('Enterprise Search', function() {
+    loadTestFile(require.resolve('./setup_guide'));
+  });
+}
```

Comment:
The difference in file directory locations confuses me a decent amount for these two different suites. Is there any particular reason for that, or can we put both of these under `x-pack/test/functional/apps/` and name one `enterprise_search` and `enterprise_search_nonconfigured` or `enterprise_search_setup` perhaps?

Reply:
Good point, making an updated now to put them in 

test/functional_enterprise_search/apps/enterprise_search/with_host_configured
test/functional_enterprise_search/apps/enterprise_search/without_host_configured

The original idea was to have the test that didn't require the host configuration to run with the rest of the functional suite.